metrics-collectd/src/main/java/com/codahale/metrics/collectd/Network.java

```diff
@@ -0,0 +1,24 @@
+package com.codahale.metrics.collectd;
+
+/**
+ * Constants from collectd/src/network.h and jcd.* property configuration.
+ */
+class Network {
+
+    public static final int TYPE_HOST            = 0x0000;
```

Comment:
I think the constants could be declared without the hexadecimal representation (it seems to me, it doesn't anything useful).